nemo/collections/nlp/modules/common/megatron_web_server.py

```diff
@@ -60,6 +62,7 @@ def get_demo(share, username, password, server_port=5555, web_port=9889, loop=No
                 repetition_penality = gr.Slider(
                     minimum=1.0, maximum=5.0, step=0.02, value=1.2, label='Repetition penalty'
                 )
+                end_strings = gr.Textbox(label="End strings (comma separated)", value="<|endoftext|>,", lines=1,)
```

Comment:
<|endoftext|> is hardcoded for the GPT2 tokenizer, we need to make this general no?

Reply:
It is used as a text representation of eos_id. Each tokenizer will use its own eos_id. So it is general.

Reply:
I think it's default value for UI, right?

Reply:
yes. it is default for text generation too